src/utils.py

```diff
@@ -22,3 +24,26 @@ def valid_date(s):
     except ValueError:
         msg = "not a valid date: {0!r}".format(s)
         raise argparse.ArgumentTypeError(msg)
+
+
+class Network(Enum):
+    MAINNET = "mainnet"
+    GNOSIS = "gnosis"
+
+    def dune_v1_repr(self) -> LegacyDuneNetwork:
```

Comment:
Nitpick: Liked the addition of the word `as` before the method name in token details. What do you think of doing the same here?